### DISCUSSION TITLE
Adapting setup for ESP32-S3 board that does not have a pin 34

### DISCUSSION THREAD

**Discussion by NotABadger** [Bodmer/TFT_eSPI#2517] [Q&A - General]
Hello there,

I'm using your beautiful library with ease. and recently switched to an ESP32-S3 with an ILI9488 8-bit parallel interface.
So I stumbled unto the Setup70d_ILI9488_S3_Parallel.h configuration file. 
I noticed that the TFT_RST definition was set to pin 34, however, there is no pin 34 on my [devkit](https://docs.espressif.com/projects/esp-idf/en/latest/esp32s3/hw-reference/esp32s3/user-guide-devkitc-1-v1.0.html).

So I set it to pin 17, and it worked just fine.
Thank you for putting the effort in this library, you're supporting a lot of hobbyists out there.

Kind regards,
Mathieu

**Comment by Bodmer** [comment]
Yes, you have to adapt the example setups according to the processor board you have and the GPIO made available.

**Comment by NotABadger** [comment]
Thank you for the answer. 
Sorry for the late response, I had two day PI planning session in the office.

I will have a more in depth look into the library in the future when I want to try different screen controllers. 
If I would like to add to the project, like e.g. trying different controllers, or do a deep dive into 16-bit bus. Can just fork and make a pull request?
Or should a discussion be opened first with the question about the changes? This is my first time taking a more in depth interest into an open-source library, but I do hold a  bachelor in technical software/
